osu.Server.Spectator/Hubs/MultiplayerHub.cs

```diff
@@ -916,12 +926,55 @@ public async Task OnMatchSettingsChanged(ServerMultiplayerRoom room)
             await Clients.Group(GetGroupId(room.RoomID)).SettingsChanged(room.Settings);
         }
 
+        public async Task<ItemUsage<ServerMultiplayerRoom>> GetRoom(long roomId) => await Rooms.GetForUse(roomId);
+
+        public async Task InternalStartMatch(ServerMultiplayerRoom room)
```

Comment:
```suggestion
        internal async Task InternalStartMatch(ServerMultiplayerRoom room)
```

This will need to be marked `internal` else it can be called by a user, which we probably don't want. Same goes for `GetRoom` above. (will need to confirm that `internal` methods are not exposed via SignalR - i'm just assuming so here).